docs/Plugins/sonarqube.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Sonarqube"
+description: >
+  Sonarqube Plugin
+---
+
+## Summary
+
+This plugin collects Sonarqube data through its REST APIs. SonarQube is a tool for static code analysis and code quality management. It can help you discover potential problems and defects in your code, and provide suggestions and solutions.
+
+## Entities
+
+Check out the [Sonarqube entities](/Overview/SupportedDataSources.md#data-collection-scope-by-each-plugin) collected by this plugin.
+
+## Data Refresh Policy
+
+Check out the [data refresh policy](/Overview/SupportedDataSources.md#sonarqube) of this plugin.
+
+## Metrics
+
+Metrics that can be calculated based on the data collected from Sonarqube:
```

Comment:
Update with: Most of SonarQube metrics are collected and can be found in DevLake's SonarQube dashboard.